packages/typescript-to-proptypes/src/createType.ts

```diff
@@ -94,7 +96,7 @@ export function createInterfaceType(init: {
 }
 
 export function createLiteralType(init: {
-  value: unknown;
+  value: string | number | ts.PseudoBigInt;
```

Comment:
This Type come from observing all the `createLiteralType()` call

two of them only take string, the last one as this typing for it's input value